sheets/api.py

```diff
@@ -1012,7 +1045,8 @@ def process_assignment_spreadsheets(self):
                 log.error(
                     "All relevant coupons have been assigned and messages have been sent, "
                     "but failed to update the spreadsheet properties to indicate status "
-                    "- %s, exception: %s" % (spreadsheet_repr(spreadsheet), str(exc))
+                    "- %s, exception: %s",
+                    (spreadsheet_repr(spreadsheet), str(exc)),
                 )
             except Exception as exc:
                 log.error(exc)
```

Comment:
This is outside the PR but should this be a `log.exception` instead, to get the stack trace?